Userland/Applications/Calendar/Calendar.cpp

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2023, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "Calendar.h"
+#include <AK/JsonParser.h>
+#include <LibConfig/Client.h>
+#include <LibFileSystemAccessClient/Client.h>
+
+namespace Calendar {
+
+Calendar::Calendar()
+{
+}
+
+OwnPtr<Calendar> Calendar::create()
+{
+    return adopt_own(*new Calendar());
+}
+
+ErrorOr<void> Calendar::add_event(JsonObject event)
+{
+    if (!m_events.has_value()) {
+        m_events = TRY(AK::JsonParser("[]"sv).parse()).as_array();
```

Comment:
Could you not just initialize a `JsonArray()` here?

https://github.com/SerenityOS/serenity/blob/87a7299078fe563b3e49f15994ee697b89d380a2/AK/JsonArray.h#L21C1-L23